test/selenium/lib/foxboxHelperSelenium.js

```diff
@@ -4,15 +4,16 @@ const path = require('path');
 
 const spawn = require('child_process').spawn;
 
+const PROFILE_PATH = path.join(process.env.HOME, '.local/share/foxbox-tests');
```

Comment:
That would explain the result travis shows now, passing the first test but not the other since it fails in getting the sign up page as the user has not been removed